locales/en/index.yml

```diff
@@ -2297,6 +2297,7 @@ bonus:
     accessibility:
       code: By pressing the item you will copy the code
     name: Carta Giovani Nazionale
+    departmentName: Dipartimento per le politiche giovanili e il servizio civile universale
```

Comment:
In english will it be the same of italian?

Reply:
@Undermaken I got no informations about its translation let's leave it in italian maybe we'll fix with a future PR